### ISSUE TITLE
Not working in FF 32.0.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#94]
```
Operating system and version (32/64bit) : Windows 7 Professional 64bit
.Net Framework version : v4.0.30
Office name and version(32/64bit) : Office 2010 64bit
Browser name and version : FireFox 32.0.1
SeleniumWrapper version : 1.0.19.0

What steps will reproduce the problem with a public website ?

In VBA when attenting to use the following script:

Dim MyFF As New SeleniumWrapper.WebDriver
Dim MyURL As String

MyURL = "https://platform.50onred.com/pops"

With MyFF
 .Start "firefox", MyURL 'This works fine and the browser windows opens'
 .Open MyURL 'Here the url is inserted and should go to the webpage, but is not.                                                         Instead I get a timeout error.'

End With

This problem is recent after the latest Firefox update. I tried previous 
versions of the Selenium Wrapper and they don't work either.


```

Original issue reported on code.google.com by `kitaraga...@gmail.com` on 16 Sep 2014 at 3:19

**Comment by GoogleCodeExporter** [comment]
```
I can confirm this is happening to me as well... had to downgrade back to 
firefox version 31 to get it working again
```

Original comment by `p...@blankshirts.com` on 16 Sep 2014 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.0.20.0
```

Original comment by `florentbr` on 18 Sep 2014 at 4:27

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 18 Sep 2014 at 4:28
- Changed state: **Fixed**
